### PR TITLE
Lock to run tests sequentially

### PR DESCRIPTION
**Pull request by mikhailOK**
Unignore a test + some small changes

### CODE REVIEW

File: test-utils/testlib/src/alphanet_utils.rs [hunk 1]
```diff
@@ -319,7 +320,7 @@ impl Drop for ProcessNode {
     fn drop(&mut self) {
         match self.state {
             ProcessNodeState::Running(ref mut child) => {
-                child.kill().unwrap();
+                let _ = child.kill().map_err(|_| println!("child process died"));
```
Comment (nearmax) [inline test-utils/testlib/src/alphanet_utils.rs]: use log

File: test-utils/testlib/src/test_locks.rs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+use std::sync::Mutex;
+
```
Comment (nearmax) [inline test-utils/testlib/src/test_locks.rs]:
```
use lazy_static::lazy_static;
```
This is old style:
```
#[macro_use]
 extern crate lazy_static;
``

File: node/tokio-utils/src/lib.rs [hunk 3]
```diff
@@ -46,6 +48,17 @@ impl ShutdownableThread {
             let _ = thread.join().expect("Error joining child thread");
         }
     }
+
+    pub fn wait_sigint_and_shutdown(mut self) {
```
Comment (nearmax) [inline node/tokio-utils/src/lib.rs]: Is `mut self` intentional? Should we explain in the docstring why do we consume `self`, is it for safety?

Comment (mikhailOK) [reply node/tokio-utils/src/lib.rs]: doesn't really matter, actually all of the methods could consume self to make their intentions more explicit
can change this to &mut self or change others to mut self

Comment (nearmax) [reply node/tokio-utils/src/lib.rs]: Whatever makes intentions more explicit.

**Pull request merged**